package.json

```diff
@@ -66,7 +66,7 @@
     "eslint-plugin-import": "2.13.0",
     "express": "4.16.2",
     "mocha": "5.0.4",
-    "nyc": "13.1.0",
+    "nyc": "^14.1.1",
```

Comment:
Is there a specific reason to update? Not complaining, just asking 😄 
Perhaps, choose a specific version (without the `^`)?

Reply:
This change was caused by ``npm audit fix``, I will revert it

Reply:
I will revert the changes for ``package.lock`` too